controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -982,8 +982,15 @@ public ModelNode getNoTextDescription(boolean forOperation) {
                 result.get(ModelDescriptionConstants.REQUIRES).add(required);
             }
         }
+
         if (getReferenceRecorder() != null) {
+            String[] capabilityPatternElements = getReferenceRecorder().getRequirementPatternSegments(name);
             result.get(ModelDescriptionConstants.CAPABILITY_REFERENCE).set(getReferenceRecorder().getBaseRequirementName());
+            if(capabilityPatternElements.length > 0) {
```

Comment:
```
if(capabilityPatternElements.length > 0 && (capabilityPatternElements.length > 1 || !name.equals(capabilityPatternElements[0])) { 
```

IOW don't output this for the default case.

If not outputting this makes the feature-spec generator too difficult let me know.